web_timeline/README.rst

```diff
@@ -56,6 +56,9 @@ the possible attributes for the tag:
 +--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | mode               | No        | Specifies the initial visible window. Available values are: 'day' to display the current day, 'week', 'month' and 'fit'. Default value is 'fit' to adjust the visible window such that it fits all items.                                                                 |
 +--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| margin             | No        | Specifies the margins around the items. It should respect the JSON format. For example '{"item":{"horizontal":-10}}'. Available values are: '{"axis":<number>}', '{"item":<number>}', '{"item":{"horizontal":<number>}}', '{"item":{"vertical":<number>}}',               |
```

Comment:
Can you mention that the options can be combined. The documentation of visjs says something different then here.

> The minimal margin in pixels between items and the time axis.

And 

> The minimal margin in pixels between items in both horizontal and vertical direction.

Can you change it to this?